variable.go

```diff
@@ -127,6 +127,16 @@ func (m *Variable) Valid() error {
 		return fmt.Errorf("invalid arguments type")
 	}
 
+	inValidNames := [11]string{"and", "import", "not", "return", "option", "test", "empty", "in", "or", "package", "builtin"}
+
+	for x := 0; x < len(inValidNames); x++ {
+		fmt.Printf(inValidNames[x])
+
+		if m.Name == inValidNames[x] {
+			return fmt.Errorf("protected variable name")
```

Comment:
return the name of the variable for clarity perhaps?
```suggestion
			return fmt.Errorf("%q is a protected variable name", inValidNames[x])
```

Reply:
Added it!